cabal.project

```diff
@@ -202,54 +161,55 @@ constraints:
   , Cabal >= 3.4.0.0
   , async-timer >= 0.2.0.0
   , unliftio-core >= 0.2.0.1
-  , cardano-api == 1.35.4
-  , cardano-node == 1.35.4
-  , generic-arbitrary == 0.2.2
+  , cardano-api >= 1.35.4
+  , cardano-node >= 1.35.4
+  , generic-arbitrary >= 0.2.2
   , iohk-monitoring >= 0.1.11
 
   -- Could probably be bumped with minor work in the wallet
-  , resource-pool == 0.2.3.2
+  , resource-pool >= 0.2.3.2
 
   -- TH Name shadowing warnings need to be addressed when bumping to 2.13.3.5
   , persistent == 2.13.3.3
 
   -- Copied from cardano-node's cabal.project:
-  , bimap >= 0.4.0
-  , libsystemd-journal >= 1.4.4
   , systemd >= 2.3.0
     -- systemd-2.3.0 requires at least network 3.1.1.0 but it doesn't declare
     -- that dependency
   , network >= 3.1.1.0
   , HSOpenSSL >= 0.11.7.2
-  , algebraic-graphs < 0.7
-  , protolude < 0.3.1
-  , cardano-prelude == 0.1.0.0
-  , base-deriving-via == 0.1.0.0
-  , cardano-binary  == 1.5.0
-  , cardano-binary-test == 1.3.0
-  , cardano-crypto-class  == 2.0.0.0.1
-  , cardano-crypto-praos  == 2.0.0.0.1
-  , cardano-crypto-tests  == 2.0.0.0.1
-  , cardano-slotting  == 0.1.0.0
-  , measures == 0.1.0.0
-  , orphans-deriving-via == 0.1.0.0
-  , strict-containers == 0.1.0.0
-  , plutus-core == 1.0.0.1
-  , plutus-ledger-api == 1.0.0.1
-  , plutus-tx == 1.0.0.0
-  , plutus-tx-plugin == 1.0.0.0
-  , prettyprinter-configurable == 0.1.0.0
-  , plutus-ghc-stub == 8.6.5
-  , word-array == 0.1.0.0
-  , word-array == 0.1.0.0
+  , cardano-prelude >= 0.1.0.0
+  , base-deriving-via >= 0.1.0.0
+  , cardano-binary >= 1.5.0
+  , cardano-binary-test >= 1.3.0
+  , cardano-crypto-class >= 2.0.0.0.1
+  , cardano-crypto-praos >= 2.0.0.0.1
+  , cardano-crypto-tests >= 2.0.0.0.1
+  , cardano-slotting >= 0.1.0.0
+  , measures >= 0.1.0.0
+  , orphans-deriving-via >= 0.1.0.0
+  , plutus-core >= 1.0.0.1
+  , plutus-ledger-api >= 1.0.0.1
+  , plutus-tx >= 1.0.0.0
+  , plutus-tx-plugin >= 1.0.0.0
+  , prettyprinter-configurable >= 0.1.0.0
+  , plutus-ghc-stub >= 8.6.5
+  , word-array >= 0.1.0.0
+
+  -- Haddock is broken in this release. Waiting for the next release
+  , http2 < 4.0.0
+
+-- Related to: https://github.com/haskell/cabal/issues/8554
+if impl(ghc == 8.10.7)
+  constraints: process == 1.6.13.2
 
 -- ----------------------------------------------------------------
 -- Flags for dependencies
 
 package cardano-config
-  flags: +systemd
+  flags: -systemd
 package cardano-node
-  flags: +systemd
+  flags: -systemd
```

Comment:
Is this ok to change?

Reply:
It just makes cardano-node not depend on systemd for logging. I'd argue depending on systemd is wrong to begin with though. Using it is a different topic but having a hard dependency on it?

All static node builds have this disabled as well. It has no functional impact.